Core/Utilities/OracleDB.py

```diff
@@ -226,10 +227,10 @@ def executeStoredProcedure(self, packageName, parameters, output=True, array=Non
       result = None
       results = None
       if array:
-        if isinstance(type(array[0]), basestring):
+        if isinstance(type(array[0]), six.string_types):
```

Comment:
This looks wrong. I think it should be:
```suggestion
        if isinstance(array[0], six.string_types):
```

Reply:
@zmathe Can you take a look? As it stands I think this can never be true as the return type of `type(...)` is always `type`.